ocean_spark/connect/hook.py

```diff
@@ -84,18 +84,18 @@ def get_app_page_url(self) -> str:
     @staticmethod
     def get_ui_field_behaviour() -> Dict:
         return {
-            "hidden_fields": ["extra", "schema"],
+            "hidden_fields": ["extra", "port"],
             "relabeling": {
                 "password": "API token",
                 "host": "Cluster id",
                 "login": "Account id",
-                "port": "Application Id",
+                "schema": "Application Id",
```

Comment:
I don't know anything about airflow, would using something called "schema" potentially have any unintended consequences or is it more or less up to us how to use those pre-existing fields?

Reply:
not that using port made sense either :D since that was fine I guess this will be fine.

Reply:
this "relabelling" approach is from the original OceanSparkHook, just fixing a bug here as port assumes an integer